pulp_rpm/app/constants.py

```diff
@@ -11,6 +11,14 @@
     (COMPRESSION_TYPES.GZ, COMPRESSION_TYPES.GZ),
 )
 
+# publication layout
+LAYOUT_TYPES = SimpleNamespace(NESTED_ALPHABETICALLY="nested_alphabetically", FLAT="flat")
```

Comment:
These names are open for debate.  Maybe just "nested" instead of "nested_alphabetically"?  What other forms of nesting would we conceivably want and does it matter enough to bother worrying about that?